tests/test_graphical_units/testing/frames_comparison.py

```diff
@@ -0,0 +1,208 @@
+import functools
+import inspect
+import itertools
+from pathlib import Path
+from typing import Any, Callable, Type
+
+from _pytest.config import ConftestImportFailure
+from _pytest.fixtures import FixtureRequest
+from _pytest.mark.structures import Mark
+
+from manim import Scene, config
+from manim._config import tempconfig
+from manim._config.utils import ManimConfig
+from manim.renderer.cairo_renderer import CairoRenderer
+
+from ._frames_testers import _ControlDataWriter, _FramesTester
+from ._test_class_makers import (
+    DummySceneFileWriter,
+    _make_scene_file_writer_class,
+    _make_test_renderer_class,
+    _make_test_scene_class,
+)
+
+SCENE_PARAMETER_NAME = "scene"
+PATH_CONTROl_DATA = Path("tests", "control_data", "graphical_units_data").absolute()
```

Comment:
Ah, this absolute path here probably causes the path issue, right? Do you want to fix this still in the scope of this PR?

There also is a typo in the variable name. :-)

```suggestion
PATH_CONTROL_DATA = Path("tests", "control_data", "graphical_units_data").absolute()
```